ee/localserver/request-controlservice.go

```diff
@@ -0,0 +1,21 @@
+package localserver
+
+import (
+	"fmt"
+	"net/http"
+)
+
+func (ls *localServer) requestControlServiceFetchHanlder() http.Handler {
```

Comment:
Nit -- typo in Handler 🙂

Reply:
Eventually, I will learn to spell Handler ...